detect_variants/vep.cwl

```diff
@@ -49,6 +49,25 @@ inputs:
             prefix: "--coding_only"
             position: 3
         default: false
+    custom_gnomad_vcf:
+        type: File?
+        secondaryFiles: [.tbi]
+        inputBinding:
+            valueFrom: |
+                ${
+                    if (inputs.custom_gnomad_vcf) {
+                        return ['--check_existing', '--custom', inputs.custom_gnomad_vcf.path + ',gnomADe,vcf,overlap,0,AF,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH']
```

Comment:
The VEP docs are not clear when if `--check_existing` is turned on by using one of the below `--*_af` options in combination with cache. I think it's fine to leave it as is since the behavior will be the same as using `--*_af` options and that's what is desired.